### ISSUE TITLE
Fix the documentation on releasing packages

### ISSUE THREAD

**Issue by newlandsvalley** [purescript-contrib/pulp#409]
The README continues to indicate that ```pulp publish``` relies on Bower exclusively,  I think this is no longer the case, because ```spago``` can now prepare the ground for publication without Bower but there is no indication of this in the Pulp documentation.

**Comment by JordanMartinez** [comment]
> because spago can now prepare the ground for publication without Bower but there is no indication of this in the Pulp documentation.

Spago can generate a `bower.json` file, but `bower` is still used to download the dependencies. See https://github.com/purescript/spago#publish-my-library=

**Comment by newlandsvalley** [comment]
I realise that, but I still think that the Pulp documentation is misleading.  It doesn't reference back to spago and can leave you with the impression that Pulp is all you need in order to publish.

**Comment by JordanMartinez** [comment]
True. Would you be up for submitting a PR that fixes this then? `pulp` is basically in maintenance mode.

**Comment by newlandsvalley** [comment]
OK - I;ll have a go.
